packages/shared/src/components/widgets/SocialShareIcon.tsx

```diff
@@ -20,22 +22,33 @@ export const SocialShareIcon = ({
   className,
   onClick,
   label,
+  iconBg,
+  pressed,
 }: SocialShareIconProps): ReactElement => {
+  const testId = `social-share-${label.replace(/\s+/g, '')}`;
+
   return (
-    <div className="flex flex-col items-center">
-      <Button
-        tag="a"
-        data-testid={`social-share-${label}`}
-        buttonSize={ButtonSize.Large}
-        href={href}
-        onClick={onClick}
-        target="_blank"
-        rel="noopener"
-        className={classNames(className, 'mb-2 text-white')}
-        iconOnly
-        icon={icon}
-      />
+    <Button
+      tag={href ? 'a' : 'button'}
+      href={href}
+      onClick={onClick}
+      target="_blank"
+      rel="noopener"
+      className={classNames(className, 'reset flex-col w-[62px] border-0')}
```

Comment:
What is this reset? also why does it feel like a custom button, not sure that's correct at this point.
Was it not our guidelined button?